source/clear-linux/tutorials/telemetry-backend/Telemetry-backend.rst

```diff
@@ -0,0 +1,493 @@
+
+   Figure 1: :guilabel:`Telemetry UI`
+
+
+Redirect telemetry records
+==========================
+
+Telemetry records from your system are sent to the server location defined in
+the :file:`/usr/share/defaults/telemetrics/telemetrics.conf` configuration
+file. You can customize this by copying this file to
+:file:`/etc/telemetrics/telemetrics.conf` and changing the ``server=``
+setting to your new server location.
+
+#. Create the :file:`/etc/telemetrics` directory and change to it
+
+   .. code-block:: console
+
+      sudo mkdir -p /etc/telemetrics
+      cd /etc/telemetrics
+
+
+#. Copy the default :file:`telemetrics.conf` file to the new
+   :file:`/etc/telemetrics` directory
+
+   .. code-block:: console
+
+      sudo cp /usr/share/defaults/telemetrics/telemetrics.conf .
+
+#. Edit the new :file:`/etc/telemetrics/telemetrics.conf` file with your
+   editor using the :command:`sudo` directive and change the
+   :guilabel:`server=` setting to ``http://localhost/v2/collector`` and save
+   this change in the new file
+
+   .. code-block:: console
+
+      server=http://localhost/v2/collector
+
+   You can also use the fully qualified domain name for your server instead of
+   localhost
+
+#. Restart the :command:`telemd` daemon to reload the configuration file:
+
+   .. code-block:: console
+
+      systemctl restart telemd
+
+Test the new telemetry backend server
+=====================================
+
+|CL| includes a telemetry test probe called :command:`hprobe` that will send a
+``hello`` record to the telemetry backend server.  To test that the telemetry
+records are now going to your new destination, run the :command:`hprobe`
+command to send a ``hello`` record to the server as follows:
+
+.. code-block:: console
+
+   hprobe
+
+The record should show up on your new server console as shown in figure 2:
+
+.. figure:: figures/telemetry-backend-2.png
+   :scale: 50 %
+   :alt: Telemetry UI
+
+   Figure 2: :guilabel:`Telemetry UI`
+
+Congratulations!  You've just set up and enabled a new telemetrics backend
+server, redirected the records from your local machine to this new server and
+tested it using the :command:`hprobe` command to send a ``hello`` record to
+it.
+
+Additional resources
+====================
+
+deploy.sh usage:
+Usage: deploy.sh -H DOMAIN [OPTIONS]
+
+Deploy snapshot of the telemetrics-backend
+
+  -a  Perform specified action (deploy, install, migrate, resetdb, restart, uninstall; default: deploy)
+  -d  Distro to deploy to (ubuntu, centos or clr; default: ubuntu)
+  -h  Print these options
+  -H  Set domain for deployment (only accepted value is \"localhost\" for now)
+  -r  Set repo location to deploy from (default: https://github.com/clearlinux/telemetrics-backend)
+  -s  Set source location (default: \"master\" branch from git repo)
+  -t  Set source type (tarball, or git; default: git)
+  -u  Perform complete uninstallation
+
+Pointer to a description of libtelemtry.so and how to incorporate it into
```

Comment:
Is this a work in progress?

Reply:
It will be once I get the time to document how to do it.  For now I will remove this sentence...I forgot I left it in there!